rpcserver.go

```diff
@@ -630,7 +630,8 @@ var _ lnrpc.LightningServer = (*rpcServer)(nil)
 // dependencies are added, this will be an non-functioning RPC server only to
 // be used to register the LightningService with the gRPC server.
 func newRPCServer(cfg *Config, interceptorChain *rpcperms.InterceptorChain,
-	implCfg *ImplementationCfg, interceptor signal.Interceptor) *rpcServer {
+	implCfg *ImplementationCfg,
+	interceptor *signal.Interceptor) *rpcServer {
```

Comment:
is the pointer change just for cleanliness? If so, should be in a separate commit (before the main commit)